packages/spectrum/tsconfig.json

```diff
@@ -32,7 +32,7 @@
   "exclude":[
     "node_modules"
   ],
-  "files": [
-    "./src/index.ts"
-  ]
+  "include": ["src/**/*", "test/**/*"],
+  "exclude": ["node_modules" ]
+
```

Comment:
Already in `master` (part of https://github.com/headwirecom/jsonforms-react-spectrum-renderers/pull/10)